src/qt/receiverequestdialog.cpp

```diff
@@ -178,12 +178,14 @@ void ReceiveRequestDialog::update()
             }
             QRcode_free(code);
 
-            QImage qrAddrImage = QImage(QR_IMAGE_SIZE, QR_IMAGE_SIZE+20, QImage::Format_RGB32);
+            QImage qrAddrImage = QImage(QR_IMAGE_SIZE + 20, QR_IMAGE_SIZE+50, QImage::Format_RGB32);
             qrAddrImage.fill(0xffffff);
             QPainter painter(&qrAddrImage);
-            painter.drawImage(0, 0, qrImage.scaled(QR_IMAGE_SIZE, QR_IMAGE_SIZE));
-            QFont font = GUIUtil::fixedPitchFont();
-            font.setPixelSize(12);
+            painter.setRenderHint(QPainter::TextAntialiasing);
+            painter.drawImage(10, 0, qrImage.scaled(QR_IMAGE_SIZE, QR_IMAGE_SIZE));
+            QFont font;
+            font.setPointSize(14);
```

Comment:
I think that won't solve the problem...
I'll open a PR for a "long term" fix.

Reply:
I forgot to upload the screenshot, but it worked for me on OSX.

Reply:
Didn't worked for me.
See #12173 which should fix it regardless of the use font / font-system.